### DISCUSSION TITLE
Progress Bar/Indicator

### DISCUSSION THREAD

**Discussion by noau** [KSXGitHub/parallel-disk-usage#263] [Ideas]
Is that possible to add a progress bar or indicator like how many files/memory has been index so it's not thought to be stuck? Like this one:
![image](https://github.com/user-attachments/assets/e5a90c04-c6e5-4698-ab8a-40771dccde4d)

**Comment by KSXGitHub** [comment]
There's `--progress` flag that shows how many files have been scanned. It wasn't on by default because it affect performance of the overall program (competing for CPU threads).

**Comment by noau** [comment]
Oh thanks for reply. It takes some time on my computer with no output so I thought it's stuck and stoped it for several times

**Reply by KSXGitHub** [reply]
Do you use an HDD? `pdu` is optimized for SSD. For HDD, there are probably faster alternatives (still slower than SSD though).

**Reply by noau** [reply]
Yep, I tried on a SSD again and it's really fast. Thanks
